app/views/admin/submissions/edit/_final_submission_fields.html.erb

```diff
@@ -7,7 +7,8 @@
 
   <%= render partial: '/admin/submissions/edit/standard_defended_at_date', locals: {f: f} if current_partner.graduate? %>
   <%= f.input :abstract, as: :text, label: 'Abstract<span class="required-asterisk">*</span>'.html_safe, input_html: { class: 'can-toggle-editability', rows: 10 } %>
-
+  <%= render partial: 'admin/submissions/abstract_help_text' %>
```

Comment:
Let's make these show only for the graduate instance for now.  If we show this for all instances, the wording might not make sense.  We can use `current_partner.graduate?` to determine if it's the graduate instance.